src/data/faq_de.json

```diff
@@ -204,6 +217,22 @@
                         "textblock": [
                             "Nein. Sowohl für Schnelltests, als auch für PCR-Tests gelten die gleichen Kriterien zur Risikoberechnung wie bisher. Falls Sie eine Risikowarnung erhalten, werden Sie in der App nicht sehen, ob diese durch das Teilen eines Schnelltest- oder eines PCR-Testergebnisses entstand. Die genauen Parameter zur Berechnung finden Sie im FAQ-Artikel <a href='#encounter_19_calc' target='_blank' rel='noopener noreferrer'>Wie erfolgt die Berechnung des Risikos mit der App (Version >=1.9)</a>."
                         ]
+                    },
+                    {
+                        "title": "Wozu dient ein Schnelltest-Profil?",
+                        "anchor": "rat_profile",
+                        "active": true,
+                        "textblock": [
+                            "Nutzer*innen können ein Schnelltest-Profil mit ihren persönlichen Daten in Form eines QR-Codes anlegen. An Teststellen teilnehmender Partner, an denen Nutzer*innen keinen Termin vorab vereinbaren müssen, wird dann lediglich der QR-Code in der App gescannt und es ist keine weitere Registrierung mehr erforderlich. So kann die Registrierung vor Ort beschleunigt werden. Die Mitarbeiter*innen an den Teststellen benötigen die persönlichen Daten, um sie im Falle eines positiven Testergebnisses an das Gesundheitsamt weiterzuleiten. Aus diesem Grund ist vor Ort immer auch ein Abgleich mit dem Personalausweis nötig. Einmal angelegt, bleibt das Profil bestehen und kann bei jedem Test wieder vorgezeigt werden."
+                        ]
+                    },
+                    {
+                        "title": "Wie kann ich ein Schnelltest-Profil anlegen, bearbeiten und entfernen?",
+                        "anchor": "rat_profile_create_edit_delete",
+                        "active": true,
+                        "textblock": [
+                            "<ul><li><b>Schnelltest-Profil anlegen:</b> Um ein Schnelltest-Profil anzulegen, können Nutzer*innen auf der Startseite ihrer Corona-Warn-App unter „Test registrieren“ auf Nächste Schritte Dort finden Sie den Bereich „Schnelltest-Profil anlegen“. Dort können sie ihre persönlichen Daten, wie Name, Adresse und Telefonnummer, eintragen. Wenn alles eingetragen ist, können sie die Daten speichern. Die Corona-Warn-App erstellt daraufhin einen QR-Code aus den Daten, den Nutzer*innen an den Teststellen vorzeigen können.</li><li><b>Schnelltest-Profil entfernen:</b> Nutzer*innen können ein Schnelltest-Profil wieder entfernen, indem sie im Schnelltest-Bereich der Corona-Warn-App auf ihr Schnelltest-Profil gehen und dort 'Schnelltest-Profil entfernen' auswählen.</li><li><b>Schnelltest-Profil bearbeiten:</b> Beabsichtig ein Nutzer*innen ein Schnelltest-Profil zu bearbeiten, kann er/sie es zunächst entfernen und danach ein neues Profil anlegen.</li></ul>"

```

Comment:
The first sentence is not a sentence in German.
Beabsichtig there is a t missing.

Reply:
2x thanks.